spec/javascripts/actions/column_action_spec.js

```diff
@@ -0,0 +1,66 @@
+import moment from 'moment';
+
+const ColumnAction = require('actions/column');
```

Comment:
What do you think using `import` here to be compliant with the rest of the code? You can do it with `import * as ColumnActions from 'actions/column';`